core/market/decentralized/src/db/dao/demand.rs

```diff
@@ -80,6 +81,22 @@ impl<'c> DemandDao<'c> {
         })
         .await
     }
+
+    pub async fn clean(&self) -> DbResult<()> {
+        // FIXME clean negotiations also
+        log::debug!("Clean market demands: start");
+        let num_deleted = do_with_transaction(self.pool, move |conn| {
+            let nd = diesel::delete(dsl::market_demand.filter(dsl::expiration_ts.lt(sql_now)))
+                .execute(conn)?;
+            Result::<usize, DbError>::Ok(nd)
+        })
+        .await?;
+        if num_deleted > 0 {
+            log::info!("Clean market demands: {} cleaned", num_deleted);
```

Comment:
a bit shorter
```suggestion
            log::info!("Cleaned {} market demands", num_deleted);
```

Reply:
hmm.. It's shorter but also incosistent with other logs:
`log::debug!("Clean market demands: start");`
`log::debug!("Clean market demands: done");`

Reply:
ok